build/configs/imxrt1020-evk/tc_protected/defconfig

```diff
@@ -0,0 +1,1081 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# TinyAra Configuration
+#
+
+#
+# Build Setup
+#
+# CONFIG_EXPERIMENTAL is not set
+# CONFIG_DEFAULT_SMALL is not set
+CONFIG_HOST_LINUX=y
+# CONFIG_HOST_OSX is not set
+# CONFIG_HOST_WINDOWS is not set
+# CONFIG_HOST_OTHER is not set
+# CONFIG_WINDOWS_NATIVE is not set
+
+#
+# Build Configuration
+#
+CONFIG_APPS_DIR="../apps"
+CONFIG_FRAMEWORK_DIR="../framework"
+CONFIG_TOOLS_DIR="../tools"
+# CONFIG_BUILD_FLAT is not set
+CONFIG_BUILD_PROTECTED=y
+# CONFIG_APP_BINARY_SEPARATION is not set
+CONFIG_BUILD_2PASS=y
+CONFIG_PASS1_TARGET="all"
+CONFIG_PASS1_OBJECT=""
+CONFIG_TINYARA_USERSPACE=0x60040000
```

Comment:
This should NOT be fixed like this. It depends on CONFIG_FLASH_PART_SIZE="256,256,4096. Otherwise user binary will be flashed based on CONFIG_FLASH_PART_SIZE="256  and this fixed value will be  used in Code. Lot of trouble. Either fix CONFIG_FLASH_PART_SIZE="256,  (make it NOT configurable) and use fixed value or calculate the start address from partititon and use it.

Reply:
I agree. But not in this PR.

Reply:
Then exclude this file from this commit. Fix only the build warning. IF tc for protected is to  be added, then it has to be fixed.

Reply:
No I don't think so. This is adding tc_protected using current implementation. If we remove that config, it does not boot up. I agree that we should modify to be set automatically from partition information. But this PR has totally different purpose.

Reply:
I did n't mean to remove the config CONFIG_TINYARA_USERSPACE. I mean remove these set of files from this commit and push after making correct changes. Why do you want to commit with a known bug?

Reply:
That is NOT a bug. It just causes inconvenient for user. It's right code of now. As I said before, to avoid mistake, we had better modify that logic. BUT not in this PR. This does NOT have that purpose. If you want to merge that with this PR, we should fix all of issue where you modify even if you don't modify. That's not good.

Reply:
This should be tracked and fixed later. For now, removed the hold.